projects/dotcms-js/src/lib/core/site.service.spec.ts

```diff
@@ -2,29 +2,30 @@ import { fakeAsync, tick } from '@angular/core/testing';
 import { Response, ResponseOptions, ConnectionBackend } from '@angular/http';
 import { MockBackend } from '@angular/http/testing';
 import { Site } from './treeable/shared/site.model';
-import { DOTTestBed } from 'src/app/test/dot-test-bed';
+// import { DOTTestBed } from 'src/app/test/dot-test-bed';
```

Comment:
remove

Reply:
This should keep comment for now, because import the DOTTestBed throw some build errors and avoid to run the dotcms-js test.